htop.c

```diff
@@ -99,6 +99,7 @@ static CommandLineSettings parseArguments(int argc, char** argv) {
    int opt, opti=0;
    /* Parse arguments */
    while ((opt = getopt_long(argc, argv, "hvmCs:td:u::Up:", long_opts, &opti))) {
+      const char *username;
```

Comment:
I'd prefer to limit the scope for `username` to only the case label that it is used in. This requires some `{}` inside the `case 'u'`, but makes it clearer that it's only ever used there. Also I kinda dislike defining an uninitialized variable that only get's initialized much later (if at all).

Reply:
Fixed